packages/core/src/connectors/mock.ts

```diff
@@ -28,35 +31,21 @@ export type MockConnectorOptions = {
-};
-
 export class MockConnector extends Connector {
   private _accounts: MockConnectorAccounts;
   private _accountIndex = 0;
-  private _options: MockConnectorOptions;
   private _connected = false;
   private _chainId: bigint = devnet.id;
+  _options: MockConnectorOptions;
```

Comment:
```suggestion
  public options: MockConnectorOptions;
```

Let's make it clear it's public.